go.mod

```diff
@@ -36,6 +36,7 @@ require (
 )
 
 require (
+	github.com/bmatcuk/doublestar/v4 v4.6.1 // indirect
```

Comment:
Just a quick comment: we need to du due diligence when adding dependencies

- Is the version compatible?
- Is it packaged in Debian?

If the surface of the API we actually use represents a small bit of the whole it might be better to copy it in.

Reply:
Separately, why is this indirect? Do we need to run "go mod tidy"?

Reply:
Looking at the package from my system:
```
zyga@novigrad:~$ LANG=C apt-cache policy golang-github-bmatcuk-doublestar-dev
golang-github-bmatcuk-doublestar-dev:
  Installed: (none)
  Candidate: 2.0.4-2
  Version table:
     2.0.4-2 500
        500 http://pl.archive.ubuntu.com/ubuntu mantic/universe amd64 Packages
        500 http://pl.archive.ubuntu.com/ubuntu mantic/universe i386 Packages
```

In Debian we have https://packages.debian.org/sid/golang-github-bmatcuk-doublestar-dev so unless we need to keep building without vendoring in Ubuntu it does look okay.

Reply:
> why is this indirect

It should not be indirect. When I added it, it marked it as indirect, but I then did some testing vs v4.6.0 and realized we needed to ensure version v4.6.1.

Go dependencies are still a bit mysterious to me (I've been spoiled by Rust's very explicit Cargo.toml, which would take `doublestar = "4.6.1"`). Is the only difference whether the ` // indirect` is at the end, or is there something else to do to mark it as a required version/dependency?

Reply:
I pushed a commit which seemed as sensible as I could make it, but the only difference is whether the ` // indirect` is included, and that I moved it to the alphabetically-ordered location.

Reply:
Just run `go mod tidy` and commit that.